src/radiation/radiation.cpp

```diff
@@ -66,13 +66,12 @@ Initialize(ParameterInput *pin, ArtemisUtils::Constants &constants, const bool d
       fluidids.push_back(n);
 
     // Control field for sparse gas fields
-    const std::string control_field = rad::opac::absorption::name();
 
     // Absorption and scattering opacity
     Metadata m = Metadata({Metadata::Cell, Metadata::Derived, Metadata::OneCopy,
-                           Metadata::Sparse, MetadataRadiation, MetadataOperatorSplit});
```

Comment:
`MetadataOperatorSplit` avoids updates in the RK-integrator steps? What does the `MetdataRadiation` do?